app/src/main/java/com/tpp/theperiodpurse/AppViewModel.kt

```diff
@@ -101,6 +102,58 @@ class AppViewModel @Inject constructor (
         userRepository.setSymptoms(sympCopy)
     }
 
+    fun updateSymptoms(symptom: Symptom): Boolean {
+        val symptoms = getTrackedSymptoms()
+        val sympCopy = symptoms.toMutableList()
+        if(symptoms.contains(symptom)){
+            sympCopy.remove(symptom)
+            setTrackedSymptoms(sympCopy as ArrayList<Symptom>)
+            return false
+        } else {
+            sympCopy.add(symptom)
+            setTrackedSymptoms(sympCopy as ArrayList<Symptom>)
+            return true
+        }
+    }
+
+    fun isSymptomChecked(symptom: Symptom): Boolean{
+        val symptoms = getTrackedSymptoms()
+        return symptoms.contains(symptom)
+    }
+
+    fun getAllowReminders(): Boolean{
+        return uiState.value.allowReminders
+    }
+
+    fun toggleAllowReminders(){
```

Comment:
When user turns off the alarm, how do you stop all previous alarms? It seems like right now once the alarm is set, it's independent from the switch, and turning it on/off doesn't do anything anymore